SoftLayer/CLI/hardware/detail.py

```diff
@@ -92,16 +92,19 @@ def cli(env, identifier, passwords, price):
 
 
 def _bw_table(bw_data):
-    """Generates a bandwidth useage table"""
+    """Generates a bandwidth usage table"""
     table = formatting.Table(['Type', 'In GB', 'Out GB', 'Allotment'])
-    for bw_point in bw_data.get('useage'):
+    for bw_point in bw_data.get('usage'):
         bw_type = 'Private'
         allotment = 'N/A'
         if bw_point['type']['alias'] == 'PUBLIC_SERVER_BW':
             bw_type = 'Public'
-            allotment = utils.lookup(bw_data, 'allotment', 'amount')
-            if allotment is None:
```

Comment:
If a server doesn't have an allotment, what does `utils.lookup(bw_data, 'allotment', 'amount')` return, I'm guessing `''` ? 

If that is the case, you can just change `if allotment is None:` to `if not allotment` to avoid having the nested if statements.

Reply:
If a server doesn't have an allotment, the  utils.lookup(bw_data, 'allotment', 'amount') returns an error.